### DISCUSSION TITLE
Add option to have the sidebar explorer only show bookmarks from files that are currently open

### DISCUSSION THREAD

**Discussion by jimjoh** [alefragnani/vscode-bookmarks#665] [Ideas]
When I traverse my bookmarks I'm only interested in bookmarks for files that are open.  Listing the bookmarks from all of my closed files just clutters the sidebar explorer for me.  I'd love an option to limit the sidebar explorer to showing just the bookmarks from open editors.

**Comment by alefragnani** [comment]
Hi @jimjoh ,

This is already being tracked in #238 . Feel free to upvote, as I'm planning to review and plan the upcoming features to be released during vacation :-)

Hope this helps

**Reply by jimjoh** [reply]
I think I've upvoted it (thumbs up on the initial post).  However, I'm not sure if it's exactly the same as my idea.  I see near the end where it there's a request to have it " display the bookmarks of the active/focused file" which I also like.  However, it'd be nice if  you could choose between current active file, all open files, all bookmarks ever created.

Anyway, thanks @alefragnani  for creating such a great extension.
